package.json

```diff
@@ -18,6 +18,7 @@
 		"@release-it/conventional-changelog": "^8.0.1",
 		"husky": "^8.0.3",
 		"release-it": "^17.0.1",
+		"scheduler": "^0.25.0",
```

Comment:
i'm not sure if this is for checking push commits  tracing etc. it was not allowing to push the changes to repo without it.